examples/docker-mixin-test/README.md

```diff
@@ -1,43 +1,30 @@
-# My First Bundle
+# Docker Mixin Example Bundle
 
-This is an empty bundle that porter has created to get you started!
+This is an example bundle to show how to use the Docker mixin in Porter!
 
 # Contents
 
 ## porter.yaml
 
 This is the porter manifest. See https://porter.sh/author-bundles/ for 
 details on every field and how to configure your bundle. This is a required
-file.
+file. 
 
-## helpers.sh
+The commands available in the docker mixin are docker pull, push, build, 
+run, remove, and login. 
 
-This is a bash script where you can place helper functions that you can call
-from your porter.yaml file.
+### Notes on docker login
```

Comment:
You've written great content here. How about we link to it from the readme in the docker login command so that people know to go here. e.g. "See an example for how to use docker login and securely provide your username and password"

Reply:
+1.  Either a "check out the example bundle and its README for a closer look at mixin functionality" somewhere in the main README, or, like Carolyn suggests, linking specifically to this section from the `login` part of the main README.